source/clear-linux/tutorials/flatpak/flatpak.rst

```diff
@@ -1,16 +1,16 @@
 .. _flatpak:
 
-Use Flatpak\* to install applications on |CLOSIA|
-#################################################
+Use Flatpak\* to install applications on Clear Linux\* OS

```

Comment:
I believe Kris set up and already merged new substitutions for the updated branding, so we should use that. This also applies to other instances in this file.